spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -136,7 +134,7 @@
     expect(page).to have_content(t('titles.idv.phone'))
   end
 
-  context 'with in person verify info controller enabled ' do
+  context 'with in person verify info controller' do
```

Comment:
You don't need this context heading anymore, since verify info controller is the only option left. I wonder if some of these specs are duplicated now too.

Reply:
I deleted one test entirely, we already checked the data so now a duplicate. I also simplified the setup to eliminate extra context block and setup. Resolved with commit f62c79c9322b77196edaa803b5e367c5f0435a1f